accepted/0000-powershell-bins.md

```diff
@@ -0,0 +1,55 @@
+# Powershell for Installed Binaries
+
+AKA Terminate Terminate Batch Job
+
+## Summary
+
+Today, installing binaries using npm creates an executable bash script and a .cmd file with the same name. Unfortunately, terminating cmd scripts with SIGINT causes a useless prompt for windows users - "Terminate batch job (Y/N)". Also, Powershell is the default shell for Windows. By creating .ps1 scripts in addition to bash and .cmd, powershell users will have a better command line experience.
+
+## Motivation
+
+Windows users have three main options when it comes to shells:
+
+- cmd: essentially unchanged from early versions of Windows
+- powershell: the default Windows shell, under active development
+- unixy shell: for example git-bash, zsh via WSL
+
+WSL is growing in popularity, but isn't super common. Many Windows users need to use or prefer using a native shell. And for those using Powershell, we can make their experience better by creating powershell binary wrappers.
```

Comment:
WSL is also Unix as far as npm is concerned, and so these shims are not created with it. It's lovely, but I agree, not really Windows support. =)